lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -130,6 +131,36 @@ def add_pods_library
           end
         end
 
+        # Add a reference to pods' resources (like xcassets) to the user target
+        #
+        # @return [void]
```

Comment:
Should that be a strict test?

I mean if I set my flag to return `true` as soon as I enter my `resource_files.each` will it be OK, or should I do more strict tests (like ensuring that this resource was not already added to the project & targets, and only return `true` if they weren't)?